### ISSUE TITLE
Signal crash when connected to internet

### ISSUE THREAD

**Issue by adrien-github** [signalapp/Signal-Android#13334]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Signal crash when connected to the network.
I got the same symptoms as in this bug report :

https://github.com/signalapp/Signal-Android/issues/13198

### Steps to reproduce
- open Signal
-  wait 2-3 seconds

**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)

If the network is connected, Signal crash. En airplane mode, Signal doesn't crash.

I tried on 2 differents smartphones, Samsung and Huawei, with the same data (I an moving from an old one to a new ont).

**Expected result** should happen after you run the steps above (i.e. what would be the correct behaviour)

### Device info

**Device:** Manufacturer Model XVI
**Android version:** 0.0.0
**Signal version:** 0.0.0

### Link to debug log

[signal-log-1703703929606.zip](https://github.com/signalapp/Signal-Android/files/13787130/signal-log-1703703929606.zip)
[signal-log-1703703934148.zip](https://github.com/signalapp/Signal-Android/files/13787132/signal-log-1703703934148.zip)

**Comment by cody-signal** [comment]
Hi @adrien-github , thanks for the bump and checking with the latest. I've raised this with the team.

**Comment by adrien-github** [comment]
Hi,

The bug is still present with the latest update of Signal, sadly.

This bug renders the usage of Signal impossible, unfortunately, for more than 2 month now.

**Comment by adrien-github** [comment]
Hi, 
It is still not working with version 6.46.7.

As I said, Signal is not usable because of this crash, so if there is anything I can do to correct it, please let me know.
 
Kind regards

**Comment by cody-signal** [comment]
Hi @adrien-github I re-raised it with the team so hopefully we can get someone on it soon. Sorry for the impact to you.

**Comment by gldhnchn** [comment]
I am also affected by this issue. I collected a adb log while starting signal in airplane mode and one while crashing. In the zip file is the signal log output.

[signal-log-1708204362316.zip](https://github.com/signalapp/Signal-Android/files/14320238/signal-log-1708204362316.zip)
[crash_log.txt](https://github.com/signalapp/Signal-Android/files/14320235/crash_log.txt)
[airplane_no_crash_log.txt](https://github.com/signalapp/Signal-Android/files/14320234/airplane_no_crash_log.txt)

Signal version: 6.46.7
Fairphone 4, Android 13

**What happend before:**
My phone stopped receiving signal messages yesterday. When I was sending messages, I received one tick for _sent_ but none for _received_ or _read_. Meanwhile my connected desktop app is still working normal. All messages I was sending from my phone were actually really sent and received by my chat partners and by my desktop app. Messages that I was sending from my desktop app, didn't appear on my phone. In the last 24h I rebooted my phone multiple times hoping that this will fix it. It didn't. Instead it became worse and signal started to crash constantly. Luckily it is still possible to open it in airplane mode, so I was able to do a backup.
